### PR TITLE
Add basic template of website.

### PR DESCRIPTION
**Pull request by LightFelicis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains Alicja's portfolio and STEP projects.
```
Comment (leon-kastler-google) [inline README.md:3]: nit pick: for later projects, you might want to split a PR into pieces that belong to together. For instance, changing the README should be separate from code changes.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <div id="greeting-container">
```
Comment (leon-kastler-google) [inline portfolio/src/main/webapp/index.html:12]: I think the indentation changed here. Did you use a different editor?

Comment (LightFelicis) [reply portfolio/src/main/webapp/index.html]: Done.

**Comment by LightFelicis** [comment]
Accidentally the commit deleting "photo-frame.css" was added in "Add About me" pull request.

**Pull request merged**